influxql/ast.go

```diff
@@ -3247,6 +3249,25 @@ func isFalseLiteral(expr Expr) bool {
 	return false
 }
 
+// SetLiteral represents a set literal.
+type SetLiteral struct {
+	Vals []string
+}
```

Comment:
I wanted this to eventually be an actual set to implement something else, but I think it's reasonable to rename this to `ListLiteral` until that change is actually done.